test/astar_bss.cc

```diff
@@ -323,11 +323,8 @@ TEST(AstarBss, test_Auto) {
       R"({"locations":[{"lat":48.859895,"lon":2.3610976338},{"lat":48.86271911,"lon":2.367111146}],"costing":"auto"})";
   std::vector<valhalla::DirectionsLeg_TravelMode> expected_travel_modes{
       valhalla::DirectionsLeg_TravelMode::DirectionsLeg_TravelMode_kDrive};
-  std::vector<std::string> expected_route{"Rue de la Perle",   "Rue Vieille du Temple",
-                                          "Rue Barbette",      "Rue Elzévir",
-                                          "Place de Thorigny", "Rue de Thorigny",
-                                          "Rue Debelleyme",    "Rue de Turenne",
-                                          "Rue Commines",      "Rue Amelot"};
+  std::vector<std::string> expected_route{"Rue de la Perle", "Rue des Archives", "Rue Pastourelle",
```

Comment:
`Rue de Thorigny` is a living street, so avoid it now

Reply:
but this is a bike share route which means its just pedestirans and bike riders, we shouldnt be avoiding living streets for those modes of travel should we?

Reply:
Yeah, but this is the test case for auto costing, as the line 270 says, so I thought this was expected